install.sh

```diff
@@ -60,4 +60,14 @@ fi
 
 echo ''
 echo 'Curator configuration complete!'
-echo '-> Note: to update the configuration, edit Documentation/config/config.env and Documentation/credentials/credentials.env'
\ No newline at end of file
+echo '-> Note: to update the configuration, edit Documentation/config/config.env and Documentation/credentials/credentials.env\n'
```

Comment:
You probably don't mean to write `\n` at the end of that string.

Reply:
Yes. It was by mistake will be fixed.